sdk/src/kalix.ts

```diff
@@ -605,22 +607,54 @@ export class Kalix {
     process.exit(0);
   }
 
+  /** @internal */
+  reportError(error: discovery.UserFunctionError): void {
+    const msg = this.reportErrorLogic(
+      error.code,
+      error.message,
+      error.detail,
+      error.sourceLocations,
+      error.severity,
+    );
+    switch (error.severity as number) {
+      case UserFunctionErrorSeverity.INFO:
+      case UserFunctionErrorSeverity.WARNING:
+        console.log(msg);
```

Comment:
There is a `console.warn` but it's an alias for `console.error`, which prints to stderr. This puts warnings on stdout with `console.log` instead, as GCP logging will mark stderr as ERRORs. But maybe having it use `console.warn` and print to stderr is better anyway?

Reply:
If we don't have any more fine grained output then I think console.warn/stderr makes sense.

Reply:
Ok, will update to use console.{info,warn,error}. And we can then switch to a proper logging layer later, with structured JSON output. Follow up issue will be https://github.com/lightbend/kalix-proxy/issues/618.